tests/system_tests/test_notebooks/conftest.py

```diff
@@ -188,6 +188,7 @@ def notebook_loader(
         kernel_name: str = "wandb_python",
         notebook_type: ipython.PythonType = "jupyter",
         save_code: bool = True,
+        skip_api_key_env: bool = False,
```

Comment:
Can you document this in the fixture docstring? For the format, I'm thinking document it as if it's the docstring for `notebook_loader`, since the fixture's result is injected into test functions, not the fixture itself.

```
"""Launch a notebook kernel.

Args:
    skip_api_key_env: ...
"""
```